src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -320,6 +327,8 @@ public void testDeleteStudent() throws InvalidParametersException, EntityDoesNot
         
       //Untested case: The deletion is not persisted immediately (i.e. persistence delay) 
       //       Reason: Difficult to reproduce a persistence delay during testing
+        
+        studentsDb.deleteStudent(s.course, s.email);
```

Comment:
Move this to before the comment up there; now it looks like "because there is an untested case, we need to delete something".